src/AutorestSwift/JsonRpc/AutorestPlugin.swift

```diff
@@ -139,8 +125,8 @@ class AutorestPlugin {
 
     func handleListInputs(response: RPCObject) {
         guard let filename = response.asList?.first?.asString else {
-            logger.log("handleListInputs filename is nil")
-            return
+            FileLogger.instance.log("handleListInputs filename is nil")
+            fatalError("handleListInputs filename is nil")
```

Comment:
```suggestion
            msg = "Unable to retrieve filename from Autorest response."
            FileLogger.instance.log(msg)
            fatalError(msg)
```

Alternatively, maybe we have a `logAndFail(msg)` helper function that always echoes the failure to the logger and then kills the process.

Reply:
Good idea.